hr_cae/models/hr_promotion.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2019 Coop IT Easy SCRL fs
+#   Manuel Claeys Bouuaert <manuel@coopiteasy.be>
+#   Robin Keunen <robin@coopiteasy.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class Promotion(models.Model):
+    _name = "hr.promotion"
+    _description = "Promotion"
+    _order = "date_start,state"
+
```

Comment:
Yes, for me, it's should be in specific repo

Reply:
I tend to agree with @quentinDupont . What do you think @mclaeysb ?

Reply:
I do agree for promotions but less so for recruitment. There would be no much left in hr_cae. What you use hr_cae modules for?

Reply:
I share @robinkeunen's opinion. I separated Promotions!